api/app/signals/settings/base.py

```diff
@@ -485,6 +493,17 @@
 
 ML_TOOL_ENDPOINT = os.getenv('SIGNALS_ML_TOOL_ENDPOINT', 'https://api.data.amsterdam.nl/signals_mltool')  # noqa
 
+# Search settings
+SEARCH = {
+    'PAGE_SIZE': 500,
+    'CONNECTION': {
+        'URL': 'elastic-index.service.consul:9200',
```

Comment:
Set via deployment? This is the first consul address making it to our hardcoded settings.

Reply:
This is how Gregg told me to do it. See BAS-2239 in Jira.